pages/post/[interviewId].js

```diff
@@ -144,19 +145,18 @@ const TitleContainer = styled.h2`
 const ThumbUpContainer = styled.button`
   display: flex;
   align-items: center;
-  margin-top: 10px;
   border: none;
   outline: none;
+  text-decoration: none;
+  background-color: #fafafa;
 `;
 const ThumbUpIcon = styled(ThumbUps)`
   width: 20px;
   height: 20px;
   margin-right: 5px;
   fill: #95091b;
-  margin-left: 13px;
-  margin-bottom: 10px;
+  margin-left: 15px;
```

Comment:
You can use the margin-inline shorthand here as well.

Reply:
Done, thank you. :)